cmd/minikube/cmd/config/validations.go

```diff
@@ -75,3 +76,11 @@ func IsValidPath(name string, path string) error {
 	}
 	return nil
 }
+
+func IsValidUrl(string, isoURL string) error {
```

Comment:
> but it not identify OS specific rule?

What do you mean? A file URI is a file URI right? It's something of the form 'file://<path>. I guess the _path_ part might differ a bit due to the drive character in Windows, but did we not have an issue around this and in the end we said that it also behaved correctly on Windows.

> I will add a generic test case around it.

+1